modules/gcloud/bigquery_test.go

```diff
@@ -78,8 +80,75 @@ func ExampleRunBigQueryContainer() {
 		}
 	}
 
-	fmt.Println(val)
+	fmt.Println(val[0])
+	// Output:
+	// 30
+}
+
+func TestBigQueryWithDataYamlFile(t *testing.T) {
+	ctx := context.Background()
+
+	absPath, err := filepath.Abs(filepath.Join(".", "testdata", "data.yaml"))
+	if err != nil {
+		log.Fatalf("failed to run container: %v", err)
+	}
+
+	bigQueryContainer, err := gcloud.RunBigQueryContainer(
+		ctx,
+		testcontainers.WithImage("ghcr.io/goccy/bigquery-emulator:0.6.1"),
+		gcloud.WithProjectID("test"),
+		gcloud.WithDataYamlFile(absPath),
+	)
+	if err != nil {
+		log.Fatalf("failed to run container: %v", err)
+	}
+
+	defer func() {
+		if err := bigQueryContainer.Terminate(ctx); err != nil {
+			log.Fatalf("failed to terminate container: %v", err)
+		}
+	}()
```

Comment:
suggestion: use `testcontainers.CleanupContainer` to simplify and ensure correct cleanup.